csharp/lang/correctness/sslcertificatetrust/sslcertificatetrust-handshake-no-trust.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+- id: correctness-sslcertificatetrust-handshake-no-trust
+  patterns:
+    - pattern-either:
+      - pattern: SslCertificateTrust.$METHOD($COLLECTION,sendTrustInHandshake=true)
+      - pattern: SslCertificateTrust.$METHOD($COLLECTION,true)
+    - metavariable-regex:
+        metavariable: $METHOD
+        regex: CreateForX509(Collection|Store)
+  fix: $METHOD($COLLECTION,false)
```

Comment:
Does this need to be `SslCertificateTrust.$METHOD($COLLECTION,false)`? I'm not sure, test locally and see what happens.

Reply:
It does, playground didn't replace correctly until I fixed this.